osu.Game.Rulesets.Osu/Difficulty/Skills/RhythmComplexity.cs

```diff
@@ -14,7 +14,7 @@ namespace osu.Game.Rulesets.Osu.Difficulty.Skills
 {
     public class RhythmComplexity : Skill
     {
-        private int circleCount;
+        private int amountHitObjectsWithAccuracy;
```

Comment:
This is a good name change, however I'm going to suggest we revert it for now, because now that lazer has accuracy for sliders too, this isn't accurate anymore.
If we want to adjust the code to include sliders if scorev2 is active, then this would be a good change.
Let's revert that that for now as `circleCount` is actually accurate, even if not semantically named.

Reply:
Reverted in 8f073b65f380476858a1de6401fcea6a0c55eee5